crates/solidity/inputs/schema/grammar/topics/05-expressions/06-operators.yml

```diff
@@ -0,0 +1,269 @@
+- name: "AssignmentOperator"
```

Comment:
I want to suggest using the literal names for these tokens, for example, `Plus` and `Equal`, instead of `Addition` and `Assignment`, and leave the semantic interpretation of these tokens to the AST. Otherwise, we run into cases like having both `Equal` (`=`) and `EqualOperator` (`==`) below.

This is also consistent with what the Typescript compiler uses, and I think it is much clearer from a user point of view when I'm using the resulting CST:

https://github.com/microsoft/TypeScript/blob/6dbec02a88156121c24a152d1e8e5e82d52fdc1d/lib/tsc.js#L7875-L7936

```js
  "{": 18 /* OpenBraceToken */,
  "}": 19 /* CloseBraceToken */,
  "(": 20 /* OpenParenToken */,
  ")": 21 /* CloseParenToken */,
  "[": 22 /* OpenBracketToken */,
  "]": 23 /* CloseBracketToken */,
  ".": 24 /* DotToken */,
  "...": 25 /* DotDotDotToken */,
  ";": 26 /* SemicolonToken */,
  ",": 27 /* CommaToken */,
  "<": 29 /* LessThanToken */,
  ">": 31 /* GreaterThanToken */,
  "<=": 32 /* LessThanEqualsToken */,
  ">=": 33 /* GreaterThanEqualsToken */,
  "==": 34 /* EqualsEqualsToken */,
  "!=": 35 /* ExclamationEqualsToken */,
  "===": 36 /* EqualsEqualsEqualsToken */,
  "!==": 37 /* ExclamationEqualsEqualsToken */,
  "=>": 38 /* EqualsGreaterThanToken */,
  "+": 39 /* PlusToken */,
  "-": 40 /* MinusToken */,
  "**": 42 /* AsteriskAsteriskToken */,
  "*": 41 /* AsteriskToken */,
  "/": 43 /* SlashToken */,
  "%": 44 /* PercentToken */,
  "++": 45 /* PlusPlusToken */,
  "--": 46 /* MinusMinusToken */,
  "<<": 47 /* LessThanLessThanToken */,
  "</": 30 /* LessThanSlashToken */,
  ">>": 48 /* GreaterThanGreaterThanToken */,
  ">>>": 49 /* GreaterThanGreaterThanGreaterThanToken */,
  "&": 50 /* AmpersandToken */,
  "|": 51 /* BarToken */,
  "^": 52 /* CaretToken */,
  "!": 53 /* ExclamationToken */,
  "~": 54 /* TildeToken */,
  "&&": 55 /* AmpersandAmpersandToken */,
  "||": 56 /* BarBarToken */,
  "?": 57 /* QuestionToken */,
  "??": 60 /* QuestionQuestionToken */,
  "?.": 28 /* QuestionDotToken */,
  ":": 58 /* ColonToken */,
  "=": 63 /* EqualsToken */,
  "+=": 64 /* PlusEqualsToken */,
  "-=": 65 /* MinusEqualsToken */,
  "*=": 66 /* AsteriskEqualsToken */,
  "**=": 67 /* AsteriskAsteriskEqualsToken */,
  "/=": 68 /* SlashEqualsToken */,
  "%=": 69 /* PercentEqualsToken */,
  "<<=": 70 /* LessThanLessThanEqualsToken */,
  ">>=": 71 /* GreaterThanGreaterThanEqualsToken */,
  ">>>=": 72 /* GreaterThanGreaterThanGreaterThanEqualsToken */,
  "&=": 73 /* AmpersandEqualsToken */,
  "|=": 74 /* BarEqualsToken */,
  "^=": 78 /* CaretEqualsToken */,
  "||=": 75 /* BarBarEqualsToken */,
  "&&=": 76 /* AmpersandAmpersandEqualsToken */,
  "??=": 77 /* QuestionQuestionEqualsToken */,
  "@": 59 /* AtToken */,
  "#": 62 /* HashToken */,
  "`": 61 /* BacktickToken */
  ```

Reply:
(done)